### PR TITLE
feat: add `ScreenOrientationService`

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

### CODE REVIEW

File: apps/demo/src/app/pages/home/home-page.component.html [hunk 1]
```diff
@@ -270,3 +270,19 @@ <h2>Notification</h2>
         class="icon"
     />
 </a>
+
+<a class="link" [routerLink]="['/', link.ScreenOrientation]">
+    <div>
+        <h2>Screen orientation</h2>
+        A library for declarative use of
+        <strong>Screen orientation API</strong>
```
Comment (waterplea) [inline apps/demo/src/app/pages/home/home-page.component.html:278]: nit: capital O )

File: libs/screen-orientation/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        "ng",
+        "screen",
+        "orientation"
+    ],
+    "homepage": "https://github.com/taiga-family/ng-web-apis/blob/main/libs/intersection-observer/README.md",
+    "bugs": "https://github.com/taiga-family/ng-web-apis/issues",
+    "repository": "https://github.com/taiga-family/ng-web-apis",
+    "license": "Apache-2.0",
+    "author": {
+        "name": "Alexander Inkin",
```
Comment (waterplea) [inline libs/screen-orientation/package.json:16]: Should be you :)

File: libs/screen-orientation/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        "email": "alexander@inkin.ru"
+    },
+    "contributors": [
+        "Roman Sedov <79601794011@ya.ru>"
+    ],
+    "peerDependencies": {
+        "@angular/core": ">=12.0.0",
+        "@ng-web-apis/common": ">=2.0.0"
```
Comment (waterplea) [inline libs/screen-orientation/package.json:24]: Shouldn't it be 3? Should we also include rxjs?

File: libs/screen-orientation/src/screen.service.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import {Inject, Injectable} from '@angular/core';
+import {WINDOW} from '@ng-web-apis/common';
+import {fromEvent, Observable} from 'rxjs';
+import {map, shareReplay, startWith} from 'rxjs/operators';
+
+@Injectable({
+    providedIn: `root`,
+})
+export class ScreenOrientationService extends Observable<ScreenOrientation> {
+    private readonly stream$ = fromEvent(this.win.screen.orientation, `change`).pipe(
```
Comment (waterplea) [inline libs/screen-orientation/src/screen.service.ts:10]: Would `fromEvent(undefined, 'change')` throw on older browsers? We should probably throw RxJS error if it's unsupported or make a `SUPPORTED` token to people can check it.

File: apps/demo/src/app/pages/home/home-page.component.html (hunk 1 above)
Comment (splincode) [reply apps/demo/src/app/pages/home/home-page.component.html]: fixed

File: libs/screen-orientation/package.json (hunk 3 above)
Comment (splincode) [reply libs/screen-orientation/package.json]: fixed

File: libs/screen-orientation/src/screen.service.ts (hunk 4 above)
Comment (splincode) [reply libs/screen-orientation/src/screen.service.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**